global_helpers/panther_iocs.py

```diff
@@ -0,0 +1,83 @@
+# SUNBURST IOCs https://github.com/fireeye/sunburst_countermeasures/blob/main/indicator_release/Indicator_Release_NBIs.csv
+# Last accessed: 12-5-2020
+SUNBURST_FQDN_IOCS = [
```

Comment:
Shouldn't we just use sets for all of these IOCs if we are checking for existence? cc @austinbyers